### ISSUE TITLE
Permissions not working

### ISSUE THREAD

**Issue by KreyZ1320rp** [TomGrobbe/vMenu#136]
I installed vMenu as instructed. I am using zap and did the One-Click install, but it didn't add "start vMenu" to the server.cfg nor did it add the exec command. I added both those as instructed. Keeping the permissions in the config folder. Added some steam IDs to the permissions and when I get in game the people added to the admin or mod group don't have permissions. Also, the menu key didn't change to F1 or 288 as I set it to.

1.  Go on game
2. Open menu with M (Set to F1)
3. Go to online players
4. See ban is missing

**Expected behavior**
I wanted to use this to setup permissions so only admins can spawn cars but civs could save cars.

![unknown-7](https://user-images.githubusercontent.com/45953914/50124583-284c3900-021a-11e9-8744-46866a31156e.png)
![unknown-8](https://user-images.githubusercontent.com/45953914/50124162-3d27cd00-0218-11e9-87cc-59e2831a0f6c.png)
![image](https://user-images.githubusercontent.com/45953914/50124198-65afc700-0218-11e9-9d51-b5de4111ca05.png)
![image](https://user-images.githubusercontent.com/45953914/50124286-c7703100-0218-11e9-9dba-2c2931e0f021.png)

*
- Server ip: 185.249.196.68:30120
- Do you have access to the files?: Yes

Hastebin isn't working for me so here is photos of my permissions.
![screenshot_13](https://user-images.githubusercontent.com/45953914/50124526-e8855180-0219-11e9-96d8-609ffe251757.png)
![screenshot_16](https://user-images.githubusercontent.com/45953914/50124535-f4711380-0219-11e9-87d1-fa0422d985cc.png)
![screenshot_17](https://user-images.githubusercontent.com/45953914/50124542-f6d36d80-0219-11e9-82b6-2f68df52b442.png)
![screenshot_18](https://user-images.githubusercontent.com/45953914/50124545-f935c780-0219-11e9-9033-8f710161b18c.png)
![screenshot_19](https://user-images.githubusercontent.com/45953914/50124546-fb982180-0219-11e9-8141-9ed9f9ab7de3.png)
![screenshot_20](https://user-images.githubusercontent.com/45953914/50124549-0357c600-021a-11e9-88df-946d8fe5e6d5.png)

**Comment by TomGrobbe** [comment]
Your server is **very** outdated.
`FXServer-master v1.0.0.679 linux`

Update to _at least_ 801 or up. ('d recommend using the latest version).
If your control panel shows you that the server is "up to date" then you (or someone else you gave access to your server) must have manually changed the server files. In which case you'll probably have to contact zap hosting via ticket support (not via live chat, they will not be able to look into this via live chat).

**Comment by KreyZ1320rp** [comment]
![image](https://user-images.githubusercontent.com/45953914/50125746-0a350780-021f-11e9-8235-ea0008d6bb74.png)

**Comment by TomGrobbe** [comment]
@KreyZ1320rp like i said, in that case you manually changed your server files and uploaded a very outdated server version. I can't fix that for you, and it is not vMenu related. However, vMenu will not work on such outdated server.

**Comment by KreyZ1320rp** [comment]
Where do you see that it is so outdated? Because I NEVER changed the server files.

**Comment by KreyZ1320rp** [comment]
I have over 300+ resources that all work, including vSync (removed for vMenu) And vMenu is the only on that causes any error.

**Comment by TomGrobbe** [comment]
>Where do you see that it is so outdated?

Right here: http://185.249.196.68:30120/info.json

Like I showed you: `FXServer-master v1.0.0.679 linux`

>Because I NEVER changed the server files.

Must have. I'm not saying you did it on purpose. Zap hosting doesn't just upload old server artifacts.

>I have over 300+ resources that all work,
That's another issue.

>And vMenu is the only one that causes any error.
Because you are using an old server. There's absolutely **nothing** I can do about this. vMenu requires features that have been added in server artifacts 801. And you're using 679. The info.json doesn't lie, period.

This is not a vMenu issue. But sadly you don't seem to be able to accept that and therefor I'm locking this issue.
